wdn/templates_4.1/scripts/load-deferred-css.js

```diff
@@ -1,25 +1,11 @@
 (function() {
-
-    var WDNLoadDeferredStyles = function() {
-        var sheet = document.createElement('link');
-        sheet.rel = 'stylesheet';
-        sheet.href = 'https://unlcms.unl.edu/wdn/templates_4.1/css/all.css?dep=$DEP_VERSION$';
-        document.getElementsByTagName('head')[0].appendChild(sheet);
-    };
-
     var WDNRemoveCriticalStyles = function() {
```

Comment:
Now that this is only being used in one spot, it doesn't make much sense for it to be its own function.